plugin-gradle/README.md

```diff
@@ -46,7 +46,7 @@ cmd> gradlew build
     -\t\t····if·(targets.length·==·0)·{
     +\t\tif·(targets.length·==·0)·{
     ...
-  Run 'gradlew spotlessApply' to fix these violations.
+  Run './gradlew :spotlessApply' to fix these violations.
```

Comment:
For the readme, I think the prior form was better.  It's smaller, and lets users ignore multi-platform/multi-project implications.

Reply:
you're probably right. I reverted this change